packages/components/grid/src/stories/scrolling.stories.ts

```diff
@@ -87,58 +106,118 @@ export const Horizontal: Story = {
           inset-block-start: 1rem;
         }
       </style>
+      <p>
+        This example shows how you can have a large grid where you can both scroll horizontally as well as vertically.
+        Here there is no <code>overflow</code> on the grid itself, but when grid detects that the rows are wider than
+        the grid itself, it will enable horizontal scrolling. Vertical scrolling is done via the page itself. Since you
+        generally don't want to horizontally scroll the page but only the contents of the grid, horizontal scrolling is
+        handled by the grid itself. It also will use the custom <code>sl-scrollbar</code> component for horizontal
+        scrolling. That ensures that the horizontal scrollbar is always visible and not just when you've scrolled to the
+        bottom of the page.
+      </p>
+      <p>
+        This example also shows how you can make the grid header sticky at an offset from the top of the viewport. When
+        you do this, you have to add a "cover" element to make sure that the rows are not visible when they've scrolled
+        past the header.
+      </p>
+      <p>
+        You can add and remove columns to show that the grid automatically adjusts the position of the custom scrollbar.
+      </p>
       <sl-button-bar>
         <sl-button @click=${onAdd}>Add column</sl-button>
         <sl-button @click=${onRemove}>Remove column</sl-button>
       </sl-button-bar>
       <div class="cover"></div>
-      <sl-grid .items=${people}>
-        <sl-grid-column header="Name" .renderer=${nameRenderer}></sl-grid-column>
+      <sl-grid .items=${students}>
+        <sl-grid-column
+          header="Student"
+          .renderer=${avatarRenderer}
+          .scopedElements=${{ 'sl-avatar': Avatar }}
+        ></sl-grid-column>
         ${columns.map(c => html`<sl-grid-column path=${c}></sl-grid-column>`)}
       </sl-grid>
     `;
   }
 };
 
-export const HorizontalOnly: Story = {
-  ...Horizontal,
-  loaders: [async () => ({ people: (await getPeople({ count: 10 })).people })]
+export const BothSticky: Story = {
```

Comment:
In Safari it looks like the column shadow is missing?

<img width="1035" alt="image" src="https://github.com/user-attachments/assets/5ba80501-b227-4182-b072-7eec3965df15" />



In Chrome it looks ok:
<img width="1095" alt="image" src="https://github.com/user-attachments/assets/e3728b75-7b2b-4de6-ae1e-04404e852849" />